File: Language/Python/ExceptionHandling.py
#ExceptionHandling
def checkAge(age) :
    if age<0 :
        raise RuntimeError("negative age not allowed")
    elif age<18 :
        print("under age")
    else :
        print("legal adult")

try :
    checkAge(21)
    checkAge(11)
    checkAge(-1)
except RuntimeError as e :
    print(e)
else :
    print("no errors found")
finally :
    print("checking done")

File: Language/JavaScript/ExceptionHandling.js
//ExceptionHandling
let checkAge=(age)=> {
    if (age<0)
    throw new Error("negative age not allowed");
    else if (age<18)
    console.log("under age");
    else
    console.log("legal adult");
}

try {
    checkAge(21);
    checkAge(11);
    checkAge(-1);
}
catch (e) {
    console.log(e);
}
finally {
    console.log("checking done");
}